app/src/main/java/com/teampophory/pophory/feature/sign_up/SignUpFirstFragment.kt

```diff
@@ -0,0 +1,114 @@
+package com.teampophory.pophory.feature.sign_up
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import androidx.core.view.isGone
+import androidx.core.view.isVisible
+import androidx.core.widget.addTextChangedListener
+import androidx.fragment.app.Fragment
+import com.tbuonomo.viewpagerdotsindicator.setBackgroundCompat
+import com.teampophory.pophory.R
+import com.teampophory.pophory.databinding.FragmentSignUpFirstBinding
+import java.util.regex.Pattern
+
+class SignUpFirstFragment : Fragment() {
+
+    private var _binding: FragmentSignUpFirstBinding? = null
+    private val binding: FragmentSignUpFirstBinding
+        get() = requireNotNull(_binding) { "앗 ! _binding이 null이다 !" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentSignUpFirstBinding.inflate(inflater, container, false)
+        binding.tvErrorMessage.isVisible = false
+        binding.btnDeleteEditText.isGone = true
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setEditText()
+        //edittext 삭제 버튼
+        deleteAllEditText()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        _binding = null
+    }
+
+    private fun deleteAllEditText() {
+        binding.btnDeleteEditText.setOnClickListener {
+            binding.editTvName.text.clear()
+        }
+    }
+
+    private fun setEditText() {
+        //텍스트창 활성화
+        binding.editTvName.apply {
+            setOnFocusChangeListener { view, b ->
```

Comment:
람다의 패러미터를 최대한 가독성 있게 적어줘...!! 그래야 다른 친구들도 볼 때 이게 어떤 기능인지 대충 알 수 있으니까!

```suggestion
            setOnFocusChangeListener { view, hasFocus ->
```